cmd/projects/locales/cmd_locates.go

```diff
@@ -0,0 +1,44 @@
+package locales
+
+import (
+	projectscmd "github.com/Smartling/smartling-cli/cmd/projects"
+	"github.com/Smartling/smartling-cli/services/projects"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	short  bool
+	source bool
+	format string
+)
+
+func NewLocatesCmd() *cobra.Command {
+	locatesCmd := &cobra.Command{
+		Use:   "locates",
+		Short: "Display list of target locales.",
+		Long:  `Display list of target locales.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			s, err := projectscmd.GetService()
+			if err != nil {
+				// TODO log it
```

Comment:
Why is the logging omitted here? Why don't you want to use the standard log module instead of the TODO comment?